app.py

```diff
@@ -237,15 +237,17 @@ def get_bundles(entry):
     root_dir = os.path.dirname(os.getcwd())
     build_dir = os.path.join("static", "build")
     asset_path = "/static/build/"
-    bundles = {"css": [], "js": []}
+    bundles = {"css": "", "js": []}
     manifest_path = os.path.join(build_dir, "assets.json")
+    css_manifest_path = os.path.join(build_dir, "styles.json")
     with open(manifest_path) as manifest:
         assets = json.load(manifest)
     entrypoint = assets["entrypoints"][entry]
     for bundle in entrypoint["js"]:
         bundles["js"].append(asset_path + bundle)
-    for bundle in entrypoint["css"]:
-        bundles["css"].append(asset_path + bundle)
+    with open(css_manifest_path) as manifest:
+        css_assets = json.load(manifest)
+    bundles["css"] = asset_path + css_assets[entry]
```

Comment:
@mailbackwards I ditched get_styles in favor of just _bundling_ with get_bundles. Is that fair and did I break any bad rules in doing so?

Reply:
That seems really nice and clean to me, but I will defer to more expert bundlers in terms of whether this breaks any rules or makes things less flexible. I didn't realize you could [access dict keys like this](https://github.com/texastribune/donations/blob/746fe9795cbc0fdf3703f9048d5672f5a9971542/templates/new_layout.html#L26) in Flask, that's neat...don't think you can do that in Django templates.

Reply:
cc: @AndrewGibson27, master of bundlers :)
I think one _could_ argue that now our CSS file names are a little more rigid, but I'm ok with that.
Oh haha well, maybe it's an antipattern? I think I thought that'd be alright to do since I was replacing something like this before...
```django
{% block stylesheets %}
  {% for stylesheet in bundles['css'] %}
    <link rel="stylesheet" type="text/css" href="{{ stylesheet }}">
  {% endfor %}
{% endblock %}
```
Would it be less surprising to see it as `{{bundles.css}}` or is it just that you're used to seeing the context give the value straight up without having to access it via key?

Reply:
I don't know of any problem with it, especially if it was already looking like that...I think it's true that you can do that in Django too with dot notation. @tjb346 might know more